### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 45

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#233] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255492-44b81d65-59eb-4619-b842-d6f13db5dabc.png)

**Comment by i0nut02** [comment]
1. (11 + 3 -1 scelgo 3-1)
2. (11 -3 + 3 -1 scelgo 3-1) --> (11 - 1 scelgo 3-1)
3. (11 + 3 -1 scelgo 3-1) - (   (11- 5 +3-1 scelgo 3-1) * 3 + (11-10 +3 -1 scelgo 3-1) * 3  - 0 ) 
per il 3 ho considerato che ogni porco può ricevere min 5 e poi distribuiamo il resto a caso , poi ho sottratto/sommato il caso in cui il porco a e b hanno entrambi 5 perle, poi il caso in cui tutti ricevono 5 perle è impossibile

**Comment by lord-marcel** [comment]
![es54](https://user-images.githubusercontent.com/36630876/152652932-3af44c97-3c81-40df-bd51-297f66d632f4.png)
